pkg/probe/http/http_test.go

```diff
@@ -73,7 +73,7 @@ func TestHTTPProbeProxy(t *testing.T) {
 
 	go func() {
 		http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
-			fmt.Fprintf(w, res)
```

Comment:
pkg/probe/http/http_test.go:76:19: printf-style function with dynamic format string and no further arguments should use print-style function instead (SA1006)

Reply:
Wouldn't `fmt.Fprint(w, res)` be the closer conversion here?

Reply:
yes, please fix.